tutorials/08_screenshot.md

```diff
@@ -0,0 +1,32 @@
+
+## Overview
+
+Screenshots of the current 3D scene can be taken using the screenshot plugin.
+The default directory for saved screenshots is in `~/.ignition/gui/pictures` and
+the filename will be the timestamp of when the screenshot was taken.
+It is possible to change the default directory to a different location, which will
+be shown in the example below.
+
+**Note**: This example is using Ignition Gazebo but is not required.
+The same steps can be done using `ign gui` with a 3D scene.
+
+First, go to the plugins menu on the top right (vertical ellipsis) then select
+'Screenshot':
+
+![Screenshot from dropdown menu](images/screenshot_menu.png)
```

Comment:
This way of adding images works on GitHub, but not after our documentation is uploaded to https://ignitionrobotics.org. We should use the `@image` syntax like [`ign-rendering` does for example](https://github.com/ignitionrobotics/ign-rendering/blame/ign-rendering4/tutorials/13_custom_scene_viewer.md#L43). 

We'll also need to add a line like this to `CMakeLists`:

`file(COPY ${CMAKE_SOURCE_DIR}/tutorials/images/ DESTINATION ${CMAKE_BINARY_DIR}/doxygen/html/images/)`

You can test the result locally following the documentation generation instructions [here](https://ignitionrobotics.org/api/gui/4.2/install.html).

Reply:
Thank you! a393787385e8b469bcaae8a5d44f8481717ed24c